Makefile

```diff
@@ -661,21 +661,18 @@ test-e2e: ## Run "docker-build" and "docker-push" rules then run e2e tests.
 	$(MAKE) docker-build docker-push \
 	test-e2e-run
 
-LOCAL_GINKGO_ARGS ?= -stream --progress
-LOCAL_GINKGO_ARGS += $(GINKGO_ARGS)
 .PHONY: test-e2e-local
 test-e2e-local: ## Run "docker-build" rule then run e2e tests.
 	PULL_POLICY=IfNotPresent MANAGER_IMAGE=$(CONTROLLER_IMG)-$(ARCH):$(TAG) \
 	$(MAKE) docker-build \
-	GINKGO_ARGS='$(LOCAL_GINKGO_ARGS)' \
```

Comment:
@furkatgofurov7 I was wrong--sorry! I thought `--progress` had been removed but it's still there in Ginkgo v2. Could you restore this with just the `--progress` flag? 🎖️

Reply:
Right, I reverted it back and left `--progress` only in the args

Reply:
@mboersma actually that looks like has been removed according to prow logs:
```
 Test Suite Failed
You're using deprecated Ginkgo functionality:
=============================================
  --progress is deprecated .  The funcitonality provided by --progress was confusing and is no longer needed.  Use --show-node-events instead to see node entry and exit events included in the timeline of failed and verbose specs.  Or you can run with -vv to always see all node events.  Lastly, --poll-progress-after and the PollProgressAfter decorator now provide a better mechanism for debugging specs that tend to get stuck. 
```

Reply:
okay, found it in the CHANGELOG, it has been removed in v2.5.0 release of Ginkgo: https://github.com/onsi/ginkgo/blob/master/CHANGELOG.md#ginkgo-output-now-includes-a-timeline-view-of-the-spec

Reply:
So it was deprecated in 2.0, removed in 2.5. Thanks for tracking this down, and sorry for my incomplete information.